keps/sig-network/1860-kube-proxy-IP-node-binding/kep.yaml

```diff
@@ -0,0 +1,22 @@
+title: Make Kubernetes aware of the load balancer behaviour
+kep-number: 1860
+authors:
+  - "@Sh4d1"
+owning-sig: sig-network
+participating-sigs:
+  - sig-cloud-provider
+reviewers:
+  - "@thockin"
+  - "@cheftako"
+  - TBD
```

Comment:
someone from one of the impacted providers?

Reply:
Please add @MorrisLaw as one of the reviewers as he reviewed the KEP and this is one of the feature requests from the DOKS team